### DISCUSSION TITLE
Installation freezes at  Using cached 7z1900.msi

### DISCUSSION THREAD

**Discussion by Jkalvein** [CGAL/cgal#7351] [Q&A]
Hi all,
I am following the installation instruction for windows in vscode,

The installation stops at these lines:

> -- Downloading https://github.com/llvm/llvm-project/releases/download/llvmorg-15.0.6/LLVM-15.0.6-win64.exe -> LLVM-15.0.6-win64.7z.exe...```
> -- Downloading https://7-zip.org/a/7z1900.msi -> 7z1900.msi...

I have 12 GB on my drive, 
why is that happening? any idea?

**Comment by sloriot** [comment]
Are you sure you are posting in the right project? I don't see any relationship with CGAL.
